### PR TITLE
Styled Conversation MessageError

### PR DESCRIPTION
**Pull request by dawinValenzuela**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://trello.com/c/mMYjbJyj/93-migrate-core-composer-primitives-to-styled-components

### CODE REVIEW

File: packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  padding: 10px;
+`;
+
+export const Placement = () => (
+  <>
+    <h1>Message placements</h1>
+    {Object.keys(MessageError.Placements).map((placement: any, i) => (
+      <div key={i}>
+        <MessageError
+          text="I'm fancy message"
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.stories.tsx:21]:
```suggestion
          text="I'm a fancy message"
```

File: packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        onErrorClick={mockOnErrorClick}
+        text="I'm a fancy message"
+      />
+    );
+    const errorText = screen.getByText('errorText').parentElement;
+    const errorIcon = screen.getByTestId('errorIcon');
+    fireEvent.click(errorText);
+    fireEvent.click(errorIcon);
+    expect(mockOnErrorClick).toBeCalledTimes(2);
+  });
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.test.tsx:49]: missing unit test for the customCss property

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.stories.tsx (hunk 1 above)
Comment (dawinValenzuela) [reply packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.stories.tsx]: done!

File: packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.test.tsx (hunk 2 above)
Comment (dawinValenzuela) [reply packages/styled-components/src/components/primitives/Chat/Conversation/MessageError/MessageError.test.tsx:49]: done!

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**